.github/hacktoberfest 2021/Guidelines 2021.md

```diff
@@ -0,0 +1,45 @@
+# Guidelines for Hactoberfest 2021 ⚡
+## Open Horizon 
+Open Horizon is a platform for managing the service software lifecycle of containerized workloads and related machine learning assets. It enables autonomous management of applications deployed to distributed webscale fleets of edge computing nodes and devices without requiring on-premise administrators.
+
+## What is Hactoberfest? 🌱
+
+Hosted by DigitalOcean for the 8th year in a row, Hacktoberfest encourages participation in giving back to the open source community by completing pull requests, participating in events, and donating to open source projects.
+
+All backgrounds and skills levels are encouraged to complete the challenge.
+
+## What is there for you in it?🤝
+- free swags 

```

Comment:
"swags" should probably be singular/plural "swag" unless you're using it intentionally incorrect ironically.